dintero-checkout-for-woocommerce.php

```diff
@@ -119,13 +113,49 @@ public function __construct() {
 		 * @return bool Whether it was successfully initialized.
 		 */
 		public function init_composer() {
-			$autoloader = DINTERO_CHECKOUT_PATH . '/vendor/autoload.php';
-			$result     = require $autoloader;
-			if ( ! $result ) {
+			$autoloader              = __DIR__ . '/vendor/autoload.php';
```

Comment:
Do we need the normal vendor autoload at all? I dont see any dependencies in the composer.json file, and no namespace declaration there for autoloading either. So could we not skip this part and just not include the vendor folder in a release? That would also make it so we dont have to rebuild the plugin during the CI script to clean dev-dependencies either.